bin/eslint-plugin-woocommerce/rules/dependency-group.js

```diff
@@ -0,0 +1,190 @@
+module.exports = {
+	meta: {
+		type: 'layout',
+		schema: [],
+	},
+	create( context ) {
+		const comments = context.getSourceCode().getAllComments();
+
+		/**
+		 * Locality classification of an import, one of "External",
+		 * "WordPress", "Internal".
+		 *
+		 * @typedef {string} WPPackageLocality
```

Comment:
Commenting can be changed here.  Also should we change this to be `WCPackageLocality` (same for namespacing the other types).

Reply:
Done in f2b7c5c2f7b095c82b1c4454c7db8699c38f87bc and 64f84a4c5f98410615e90585dce8238fe6b6b84a.